daemons/leader_stats_update_daemon.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+require_relative "../config/environment"
+
+@leaders_limit = 6
+@fetched_leaders_limit = 20
+@mainnet = "mainnet"
+@networks = [@mainnet, "testnet"].freeze
+sleep_time = 2 # seconds
+
+def all_leaders
```

Comment:
perhaps it would be a good idea to move all the methods to separate module in concerns? It would be a bit less messy.

Reply:
Sure, I've placed this logic in daemons/concerns.